From f067ed761db4df999993af9280030b5df76dc313 Mon Sep 17 00:00:00 2001
From: YAEGASHI Takeshi <yaegashi@debian.org>
Date: Fri, 27 Jan 2017 04:21:42 +0900
Subject: Enable Ozone platform dispmanx

dispmanx is a native graphics library supporting OpenGL ES/EGL
for Broadcom VideoCore IV on Raspberry Pi.
---
 ui/ozone/ozone_extra.gni | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/ui/ozone/ozone_extra.gni b/ui/ozone/ozone_extra.gni
index c46a90b..fe28f5f 100644
--- a/ui/ozone/ozone_extra.gni
+++ b/ui/ozone/ozone_extra.gni
@@ -5,13 +5,13 @@
 # This list contains the name of external platforms that are passed to the
 # --ozone-platform command line argument or used for the ozone_platform build
 # config. For example ozone_external_platforms = [ "foo1", "foo2", ... ]
-ozone_external_platforms = []
+ozone_external_platforms = ["dispmanx"]
 
 # This list contains dependencies for external platforms. Typically, the Ozone
 # implementations are placed into ui/ozone/platform/ and so this will look
 # something like:
 # ozone_external_platform_deps = [ "platform/foo1", "platform/foo_2", ... ]
-ozone_external_platform_deps = []
+ozone_external_platform_deps = ["platform/dispmanx"]
 
 # If a platform has unit tests, the corresponding source_set can be listed here
 # so that they get included into ozone_unittests.
-- 
2.9.3

